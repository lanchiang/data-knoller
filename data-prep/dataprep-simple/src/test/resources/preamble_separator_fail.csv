1,2,5,6
3,4,7,8
 21,2,45,6
33,43,7,38
#postamble,,,
#postamble
12,43,5,1
,32,45,76,,
,,#postamble
,,,fake