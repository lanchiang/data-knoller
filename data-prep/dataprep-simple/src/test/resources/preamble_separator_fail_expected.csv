1,2,5,6
3,4,7,8
 21,2,45,6
33,43,7,38
#postamble,,,
12,43,5,1
,,#postamble
,,,fake